### DISCUSSION TITLE
Offer a simple cli mode to post an image plus caption to a user/group/public channel/private channel

### DISCUSSION THREAD

**Discussion by lightaffaire** [d99kris/nchat#271] [Ideas]
I currently use telegram-cli to upload images+captions to a few channels but it is no longer supported and it is pretty flakey.

Could nchat be expanded to offer a very basic cli mode?

ie. 

nchat --list-[users|groups|channels]

nchat --send-to CHANNEL --msg TEXT

nchat --send-to CHANNEL --image IMAGE --caption TEXT

lots of people are looking for a replacement for old telegram-cli installs 

iain

**Comment by d99kris** [comment]
Hi @lightaffaire - this request has [come up before](https://github.com/d99kris/nchat/discussions/212), but it's unfortunately out of scope for this project.
